plugins/UsersManager/angularjs/personal-settings/personal-settings.controller.js

```diff
@@ -43,6 +43,40 @@
             this.doesRequirePasswordConfirmation = true;
         };
 
+        this.signupForNewsletter = function () {
+            var signupBtn = $('#newsletterSignupBtn');
+            signupBtn.html('Loading...');
+
+            var ajaxHandler = new ajaxHelper();
+            ajaxHandler.addParams(
+                {module: 'UsersManager', action: 'newsletterSignup'}, '' +
+                'GET'
+            );
+
+            var errorCallback = function() {
+                debugger;
+                $('#newsletterSignupMsg').hide();
+                $('#newsletterSignupFailure').show();
+                signupBtn.html('Try Again');
```

Comment:
This + loading text should be internationalized, correct?

Reply:
Changed